src/generic/course-upload-image/index.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import _ from 'lodash';
+import last from 'lodash';
```

Comment:
```suggestion
import { last } from 'lodash';
```
or `from 'lodash-es';`